### DISCUSSION TITLE
Day 18

### DISCUSSION THREAD

**Discussion by iminavo** [iminavo/scrum#9] [Show and tell]
4 May 2022, Wednesday

Phillip does not understand what are we estimating given the constantly changing reality, and local tech leads think about how can they _sell_ this /fact/ to Zoey.

# Local standup

We've noticed that the amount of open PRs on a particular service may not be resolved until the end of the sprint. We tried to look into possible solutions but did not come up with anything specific. The discussion was good, though.

> Vitar put Dan in charge of PR review\merge orchestration, to minimize the number of collisions.

I am not sure Vitar understands how code reviews work.

# Grooming

> A piece of functionality was modified due to new insights.

To my surprise, the tech leadership was a bit surprised. Naturally, estimated complexity points for the task are now invalid. We need to understand how to handle such cases, since hoping that functionality won't change is naive.

> We've had 8 grooming sessions till now.

I have never seen a _proper_ one.

We moved from grooming tasks to technical review tasks. The good thing is we use the meetings as we find fit. The bad thing is the meetings are defined by our processes and do not serve their purpose.

# Global standup

There were the status updates. Then we started going over the PR on one of our services, which was very informative.

Forrest asked for a demo from Tolegen about tapir. 

I find Phillip more and more reasonable. It turns out he has an order from his management, so only he can merge the PRs. Nevertheless, something has to be done to enhance the codeflow in our repo. An approach, similar to git-flow was proposed, but Phillip did not like it since he had a bad experience with it. He acknowledges the downfalls to trunk-based development but thinks it is worth it. 

I'm willing to see how this will work and I think we can come up with a comfortable flow over time.

**Comment by f-person** [comment]
>We've had 8 grooming sessions till now.
>>I have never seen a _proper_ one.

:)))
